.github/actions/security-issues/action.yml

```diff
@@ -15,6 +15,10 @@ inputs:
     description: 'Github Token'
     required: true
 
+  project:
+    description: 'Project the created tickets shall be associated with'
```

Comment:
Add an example. Is this the project ID or name?

Reply:
Added the information to the documentation, see below.